src/ui/components/shared/LoadingScreen.tsx

```diff
@@ -9,11 +9,27 @@ import { useAppSelector } from "ui/setup/hooks";
 import { DefaultViewportWrapper } from "./Viewport";
 import styles from "./LoadingScreen.module.css";
 
-export default function LoadingScreen({ message }: { message: ReactNode }) {
+export default function LoadingScreen({
+  footer,
+  message,
+}: {
+  footer?: ReactNode;
+  message: ReactNode;
+}) {
   const isAwaitingSourceMaps = useAppSelector(getAwaitingSourcemaps);
   const uploadingInfo = useAppSelector(getUploading);
 
-  const showHighRiskWarning = useHighRiskSettingCount() > 0;
+  const highRiskSettingsActive = useHighRiskSettingCount() > 0;
+
+  if (!footer) {
+    if (highRiskSettingsActive) {
```

Comment:
nit: those could be merged

Reply:
I don't feel strongly but I often feel like loosely related conditionals read more clearly when explicitly separated 😆 I'm noticing a pattern of you preferring them merged

Reply:
If this is an intentional pattern on ur end then i don't mind it 👍